docs/operator-guide/widgets/krakend.md

```diff
@@ -0,0 +1,91 @@
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# KrakenD Integration
+This documentation provides comprehensive guidance on integrating KrakenD into KubeRocketCI. Acting as an API gateway, it securely forwards tokens like those for SonarQube and Dependency-Track while keeping them hidden from the client side. This ensures that only authorized access is granted to backend services without exposing credentials, enhancing KubeRocketCI’s data protection standards and preventing unauthorized data leaks.
```

Comment:
remove details on:
"""
Acting as an API gateway, it securely forwards tokens like those for SonarQube and Dependency-Track while keeping them hidden from the client side. This ensures that only authorized access is granted to backend services without exposing credentials, enhancing KubeRocketCI’s data protection standards and preventing unauthorized data leaks.
"""